src/views/Header.js

```diff
@@ -279,26 +277,23 @@ class Header extends React.Component<void, HeaderProps, void> {
         hasRightComponent: !!rightComponent,
       });
 
-      appBar = (
-        <View style={styles.appBar}>
-          {titleComponent}
-          {leftComponent}
-          {rightComponent}
-        </View>
-      );
+      children = [
+        titleComponent,
+        leftComponent,
+        rightComponent
+      ];
     }
 
     return (
-      <Animated.View {...rest} style={[styles.container, style]}>
-        {appBar}
+      <Animated.View {...rest} style={[styles.container, {marginTop: STATUSBAR_HEIGHT, height: APPBAR_HEIGHT}, style]}>
```

Comment:
Okay, I submitted a fix for this, which uses additional child component with `flex: 1`.